conda/recipes/cuvs-bench/recipe.yaml

```diff
@@ -57,13 +57,14 @@ requirements:
     - rmm =${{ minor_version }}
     - scikit-learn>=1.5
     - if: linux64
-      then: mkl =2023
+      then:
+        - libaio  # Required for diskann_ssd at runtime
```

Comment:
Is `libaio` also needed to build the package? Do we need an entry in `host` for that?

The entry in `ignore_run_exports` wasn't doing anything because no run exports were being added. Run exports would only be present if `libaio` was in `host` to begin with.

Reply:
Looks like libaio is in `host` for `cuvs-bench-cpu`, right?